lib/addax-rdbms/src/main/java/com/wgzhao/addax/rdbms/util/DataBaseType.java

```diff
@@ -50,7 +50,15 @@ public enum DataBaseType
     Sybase("sybase", "com.sybase.jdbc4.jdbc.SybDriver"),
     Databend("databend", "com.databend.jdbc.DatabendDriver"),
     Access("access","net.ucanaccess.jdbc.UcanaccessDriver"),
```

Comment:
We cannot list all the database JDBC driver classes in this class. For drivers not specified in this class, you can resolve this by specifying the `driver` configuration item in the job configuration file. For more details, please refer to https://wgzhao.github.io/Addax/latest/reader/rdbmsreader/

Reply:
Because the setDriverClass method of the DataBaseType enum class can cause errors from rdbmsreader to rdbmswriter, I have to add drivers for as many database types as possible.